src/Sylius/Component/Currency/spec/Model/CurrencySpec.php

```diff
@@ -91,6 +91,17 @@ function it_can_set_enabled_value()
         $this->shouldNotBeEnabled();
     }
 
+    function it_is_not_base_currency_by_default()
+    {
+        $this->isBase()->shouldReturn(false);
+    }
+
+    function it_can_can_be_base_currency()
+    {
+        $this->setBase(true);
+        $this->isBase()->shouldReturn(true);
```

Comment:
IMHO the current code is more verbose and easier to understand than `shouldBeBase()`.